test/commands/solution/search_via_representations_test.rb

```diff
@@ -88,58 +85,37 @@ class Solution::SearchViaRepresentationsTest < ActiveSupport::TestCase
   end
 
   test "filter: criteria" do
-    user = create :user, handle: 'john'
```

Comment:
I'm removed lots of duplication and unnecessary things like this ^^. I've also added a `create_solution` method that simplifies things. This whole test file could do with being updated to use that based on the work I've done here. I found there was so much noise in the setup it was hard to find the signal of what was unique about each test etc.